apps/01PigLatin.js

```diff
@@ -6,8 +6,87 @@ prompt.start();
 
 
 function pigLatin(word) {
+  // Concept Check
+ //
+ // Please explain why we are using String.prototype.toLowerCase() here.
+ // For more info:
+ // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+ //
+ // Your Answer:
+ //replaces any upper case letter with lowercase so less lines are required to accomodate them.
+ word  = word.toLowerCase();
 
-    // Your code here
+ // Concept Check
+ //
+ // What is the significance of setting vowelIndex to -1?
+ // This might help:
+ // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+ //
+ // Your Answer:
+ //used to check if a word starts with a vowel, if so another action takes place. Also to determain where the first vowel is so the app knows where to cut a word.
+ var vowelIndex = -1; // Set it to assume there are no vowels
```

Comment:
We are not really checking here to see if a word starts with a vowel, just where the first vowel is located.